playbooks/10 - SP - FortiSOAR Health Assessment/FortiSOAR Health Assessment.json

```diff
@@ -0,0 +1,334 @@
+            "targetStep": "\/api\/3\/workflow_steps\/38becf07-595d-4aae-ae16-7f4b4922576b",
+            "sourceStep": "\/api\/3\/workflow_steps\/7775968f-86e7-4016-9192-4e1afecac4c3",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "cb970b34-1413-46ec-849a-a1810acf650a"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Generate YAML File -> Generate MD",
+            "targetStep": "\/api\/3\/workflow_steps\/a0dbccd2-7a6c-43db-86df-a5b53fcaa42d",
+            "sourceStep": "\/api\/3\/workflow_steps\/38becf07-595d-4aae-ae16-7f4b4922576b",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "369ea352-64b0-47fa-b557-d68f35b04645"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Get Current System Configuration -> Get FortiSOAR Information",
+            "targetStep": "\/api\/3\/workflow_steps\/872423de-9dba-4743-b001-cba72adfcea8",
+            "sourceStep": "\/api\/3\/workflow_steps\/2b9079b1-d32b-4fc7-bdcb-d82d0813255a",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "d8a9871b-1090-43d6-b3f1-3ca5b4c0bdd5"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Get FortiSOAR Information -> Generate Recommnded Configuration",
+            "targetStep": "\/api\/3\/workflow_steps\/c3ec3fd5-06a8-40fe-81d1-0ac8d143f9b3",
+            "sourceStep": "\/api\/3\/workflow_steps\/872423de-9dba-4743-b001-cba72adfcea8",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "4c301e2b-1e2c-47c0-b6ae-fdf24142a23f"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Get System Configuration -> Get Current System Configuration",
+            "targetStep": "\/api\/3\/workflow_steps\/2b9079b1-d32b-4fc7-bdcb-d82d0813255a",
+            "sourceStep": "\/api\/3\/workflow_steps\/315f7668-0b41-47d0-b471-0c514013ea1a",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "2091977a-2da9-4e1b-bd9a-e0d1d64eb17d"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Start -> Get System Details",
+            "targetStep": "\/api\/3\/workflow_steps\/315f7668-0b41-47d0-b471-0c514013ea1a",
+            "sourceStep": "\/api\/3\/workflow_steps\/202b919b-438f-4738-8b11-48270958528a",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "d31774c7-08e2-4b8c-b0f5-91f30fe9df6b"
+        }
+    ],
+    "groups": [],
+    "priority": "\/api\/3\/picklists\/2b563c61-ae2c-41c0-a85a-c9709585e3f2",
+    "uuid": "7f5f86e0-56bd-47da-8b07-9dfc48e4d402",
+    "owners": [],
+    "isPrivate": false,
+    "deletedAt": null,
+    "recordTags": [
+        "FortiSOAR Health Assessment"
```

Comment:
@cs-rushikesh-mehakare  as discussed, Add suitable tags "ManualTrigger" or "Referenced" to all the playbooks

Reply:
Done